spec/lang/step/operators.md

```diff
@@ -250,6 +250,54 @@ impl<M: Memory> Machine<M> {
 }
 ```
 
+### Raw Equality
+```rust
+impl<M: Memory> Machine<M> {
+    fn load_raw_data(&mut self, ptr : Pointer<<M as Memory>::Provenance>, ptr_ty : PtrType) -> Result<List<u8>> {
+        let PtrType::Ref { mutbl, pointee } = ptr_ty else {
+            panic!("non-reference input to raw equality");
+        };
+        if mutbl == Mutability::Mutable {
+            panic!("mutable reference to raw equality");
+        }
```

Comment:
I don't see why we should reject this? Seems fine if people give us mutable references.